specs/0102-terminology-translation.md

```diff
@@ -0,0 +1,29 @@
+- Feature Name: Terminology Translation
+- Created: 2020-04-24
+- Associated Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1381959
+
+# Summary
+
+Add the ability to translate terminology.
+
+# Motivation
+
+As part of [bug 1624557](https://bugzilla.mozilla.org/show_bug.cgi?id=1624557) we've added the ability to store terminology in the database and use term translations during the translation process. However, localizers currently don't have the ability to translate terms, which means they only partially benefit from using terminology. They can see what terms mean and how they can be used in source strings, but since they aren't translated, translations need to be manually looked for.
+
+# Out of scope
+
+- **Terminology creation**: the ability to manually add, remove and review terms.
+- **VCS integration**: the ability to sync terminology with VCS.
+- **Checks**: the ability to check if translations include term translations.
+
+# Feature explanation
+
+We need a new built-in translation project (called "Terminology"), which will allow for translation of all terms using the translation workbench. The project would be similar to system projects (e.g. "Tutorial") in that it would be enabled automatically for any locale and any newly added locale, but would (unlike system projects) remain visible in dashboards and included in team stats.
+
+For every `Term` with the `do_not_translate` and `forbidden` field set to False, an `Entity` will be created in the "Terminology" project with the following data:
```

Comment:
Should `forbidden` be localizable? @gueroJeff?

Reply:
Within this scope, `forbidden` is only useful for source content authors. This is one area where n:n mapping comes into play. For now, the entries as a whole should be `forbidden` so that they don't enter the pipeline. We can figure out the mapping in a different iteration.